v1/dns/index.ts

```diff
@@ -13,14 +18,22 @@ export async function initDnsServer (port: number, store: SiteConfigStore, logge
       const trimmedName = name.replace('_dnslink.', '')
       store.get(trimmedName)
         .then(({ links }) => {
-          // TODO(support http and hyper)
           if (links.ipfs !== undefined) {
             response.answers.push({
               name,
               type: dns2.Packet.TYPE.TXT,
               class: dns2.Packet.CLASS.IN,
               ttl: 60,
-              data: `dnslink=${links.ipfs.link}`
+              data: `dnslink=${protocolToMultiformat(links.ipfs.pubKey)}`
+            })
+          }
+          if (links.hyper !== undefined) {
+            response.answers.push({
+              name,
+              type: dns2.Packet.TYPE.TXT,
+              class: dns2.Packet.CLASS.IN,
+              ttl: 60,
+              data: `dnslink=${protocolToMultiformat(links.hyper.link)}`
```

Comment:
```suggestion
              data: `dnslink=${protocolToMultiformat(links.hyper.raw)}`
```

Should just be using the raw public key url. Lets assume that `{protocol}.link` will always have the domain in it